<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="Display">
    <attribute name="title">
     <string>Display</string>
    </attribute>
    <widget class="QCheckBox" name="flip_hor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip image horisontally</string>
     </property>
    </widget>
    <widget class="QSlider" name="contrast">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="contrast_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
    <widget class="QLabel" name="contrastValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="flip_vert">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Flip image vertically</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Lighting">
    <attribute name="title">
     <string>Lighting</string>
    </attribute>
    <widget class="QGroupBox" name="color">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QSlider" name="color_R">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="color_G">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="color_B">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="color_R_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QLabel" name="color_G_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="color_B_label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="direction">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>171</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <widget class="QSlider" name="direction_Z">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="direction_Y">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSlider" name="direction_X">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="direction_Z_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="direction_Y_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="direction_X_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Intensity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Intensity</string>
     </property>
     <widget class="QSlider" name="diffuse">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>16</width>
        <height>160</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="diffuse_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Kinect">
    <attribute name="title">
     <string>Kinect</string>
    </attribute>
    <widget class="QSlider" name="minHeight">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>20</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="maxHeight">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>20</width>
       <height>261</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="resolution">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>320x240</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>640x480</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="resolution_label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
    <widget class="QLabel" name="minHeight_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum height</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxHeight_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum height</string>
     </property>
    </widget>
    <widget class="QLabel" name="minHeightValue">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="maxHeightValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Skybox">
    <attribute name="title">
     <string>Skybox</string>
    </attribute>
    <widget class="QPushButton" name="loadSkyboxLeft">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgSkyboxRight">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSkyboxBack">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgSkyboxTop">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSkyboxTop">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSkyboxFront">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgSkyboxLeft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imgSkyboxBack">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSkyboxRight">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sea</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sunset</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="imgSkyboxFront">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imgSkyboxBottom">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>160</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSkyboxBottom">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>280</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Textures">
    <attribute name="title">
     <string>Textures</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Planet Earth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fantastic planet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="waterImg">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sandImg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="grassImg">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="stoneImg">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="snowImg">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>140</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadWater">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSand">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadGrass">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadStone">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>260</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadSnow">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>260</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Terrain">
    <attribute name="title">
     <string>Terrain</string>
    </attribute>
    <widget class="QSlider" name="waterMax">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sandMax">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="grassMax">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="stoneMax">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>waterMax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>sandMax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>grassMax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>stoneMax</string>
     </property>
    </widget>
    <widget class="QLabel" name="waterMaxLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="sandMaxLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="grassMaxLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="stoneMaxLabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>250</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset to default</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
